uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-files-tab.js

```diff
@@ -85,10 +87,24 @@ define([
     };
 
     (function init () {
-      vm.supportedFileTypes = _.keys(sharedSettings.fileUploader.allowedMimeTypes);
+      loadSupportedFileTypes();
```

Comment:
Since the file types are now async, it is now possible to see the UI without file types to be loaded. In this case you will see an unfinished sentence and/or it may induce more issues in the future. I recommend to use a spinner on the whole block (which is inside a dashed border) until all data is loaded. This will guarantee a stable UI.

cc @AkA84